### DISCUSSION TITLE
Nepharite Of Algeroth (ALGEROTH) [R]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#15] [Card Balance]
Digital card game :
Nepharite Of Algeroth (ALGEROTH) [R]
NEPHARITE, POWERIMMUNITY
Your other ALGEROTH Warriors gain +2 to [F] and [A] when they attack (but not when attacked).
F: 9 S: 5 A: 4 V: 9

My suggestion :
NEPHARITE,
May be bestowed with DARKSYMMETRY cards with no Action cost during combat
F: 9 S: 5 A: 7 V: 9
![](https://cdn.discordapp.com/attachments/783792005733285898/828223274277535744/unknown.png)

**Comment by TomekPL-1** [comment]
I'm OK with his current ability "NEPHARITE, POWERIMMUNITY Your other ALGEROTH Warriors gain +2 to [F] and [A] when they attack (but not when attacked)."

But his stats make him very difficult to use. Very low A with high V makes him perfect catch for opponent.

**Reply by 21-ric** [reply]
What's your suggestion about stats? "Nepharite Warlord"should be more powerfull than "nepharite of Algeroth".
About POWERIMMUNITY, does it match with the lore?

**Reply by TomekPL-1** [reply]
I would suggest 8:F 3:S 7:A 9:V

Your version F: 9 S: 5 A: 7 V: 9 is also OK.

**Reply by 21-ric** [reply]
If we have a "Warlord Nepharite" for 9[DP], we could have a "Nepharite of Algeroth" for 8[DP]

Nepharite Warlord (ALGEROTH) [R]
NEPHARITE
At the begginning of your turn, create an Undead Legionnaires token into your Kohort. If you can't,
your Undead Legionnaires gain +1 to [F] and [A] .
F: 9 S: 6 A: 8 V: 9

Nepharite Of Algeroth (ALGEROTH) [R]
NEPHARITE, 
Your other ALGEROTH Warriors gain +2 to [F] and [A] when they attack (but not when attacked).
F: 9 S: 5 A: 7 V: 8

**Comment by Kite80** [comment]
I like those ideas but don't give too much A to nepharites, they are big targets and this would force players to shield them from powerful attackers

**Comment by Sol-au-fein** [comment]
Nepharite of Algeroth:
7/5/6/7 
Gain +1/+1/+1 for each dark symmetry gift bestowed to him.

**Reply by 21-ric** [reply]
Nice version. 👍
